coins.json

```diff
@@ -1206,6 +1206,31 @@
       "clientDocs": "https://docs.harmony.one/home/harmony-networks/harmony-network-overview/mainnet"
     }
   },
+  {
+    "id": "Oasis",
+    "name": "Oasis",
+    "coinId": 474,
+    "symbol": "ROSE",
+    "decimals": 9,
+    "blockchain": "OasisNetwork",
+    "derivationPath": "m/44'/474'/0'/0/0",
```

Comment:
```suggestion
    "derivationPath": "m/44'/474'/0'/0'/0'",
```

Reply:
What is it about?

Reply:
Oasis requires that all items in the derivation path are hardened

Reply:
Ok. Accepted!

Reply:
could you please add one test to `TWHDWalletTests.cpp`? match ledger's address